assets/js/components/file-labels.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import _ from "lodash";
+
+class FileLabels extends React.Component {
```

Comment:
Since this doesn’t have any internal state that it’s using, could this just be a stateless / functional component instead? https://reactjs.org/docs/components-and-props.html

Reply:
@dgalarza I guess I'm not sure of the difference, even after reading that link 😕